frontend/utils/time.js

```diff
@@ -44,12 +44,20 @@ export function compareISOTimestamps (a: string, b: string): number {
   return A > B ? 1 : (A < B ? -1 : 0)
 }
 
+export function lastDayOfMonth (date: Date): Strin {
+  return new Date(date.getFullYear(), date.getMonth() + 1, 0)
+}
+
 // TODO: Provide locale fallback in case navigator does not exist (e.g. server, Mocha, etc...)
 const locale = (typeof navigator === 'undefined' && 'en-US') || (navigator.languages ? navigator.languages[0] : navigator.language)
 
 export function humanDate (
-  datems = Date.now(),
+  datems,
   opts = { month: 'short', day: 'numeric' }
 ) {
+  if (!datems) {
+    console.error('humanDate:: 1st arg `datems` is required')
+    return ''
+  }
```

Comment:
If it's required, it should throw `GIErrorUIRuntimeError`?

```js
if (!date) {
  throw new GIErrorUIRuntimeError('humanDate: missing first argument')
}
```

Also consider renaming from `datems` to `date` since it's usually a string that's passed in.

Reply:
throwing an error would break the application because we don't have try/catch for each time we use `time.js`. I think that would be a very drastic outcome from a simple util fn.

Reply:
OK, fair enough 👍